src/Hl7.Fhir.Support.Poco/Utility/FhirVersionUtility.cs

```diff
@@ -44,6 +44,25 @@ public static FhirRelease Parse(string version)
             };
         }
 
+        /// <summary>
+        /// Converts a version number into a specific official FHIR Release
+        /// </summary>
+        /// <param name="version">Fhir Release version number</param>
+        /// <param name="release">Official FHIR Release</param>
+        /// <returns>true if the conversion succeeded; false otherwise.</returns>
+        public static bool TryParse(string version, out FhirRelease? release)
+        {
+            release = null;
+            try
+            {
+                release = Parse(version);
+                return true;
+            }
+            catch(NotSupportedException)
```

Comment:
This is exactly the wrong way around ;-)  Since throwing exceptions is expensive (with all the stack unwinding etc), it's better to have the main implementation of the parser in TryParse, and then base the Parse on top of TryParse (so it throws when TryParse returns false)!

Reply:
Ahhh fair enough. Will fix!